data/src/main/java/org/oppia/data/backends/gae/NetworkSettings.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.data.backends.gae
+
+object NetworkSettings{
+
+  const val BASE_URL = "https://oppia.org"
```

Comment:
Please document this & next comment.

Also, please add a TODO on #74 to move these to the DI graph.

Reply:
I have added a TODO comment. Not sure if you wanted me to do something else. So please check this once again.